vms/mac/create-macos-bootable-for-virtualbox.sh

```diff
@@ -0,0 +1,31 @@
+#! /bin/bash
+
+# Installer disk created by this script was succesfully tested on Virtualbox 6.1.4 r136177.
+
+# First argument to the program should be the absolute path to the macOS installer app.
+# Example: "/Applications/Install macOS Catalina.app"
+APP_PATH=$1
+
+# Installer name is used for the volume name and disk image names.
+INSTALLER_NAME="macOSinstaller$(date +%d%m%Y)"
+
+# Make an image to create the bootable media in /tmp.
+# Make sure the size is enough for future images. 9000m is enough for Catalina 10.15.4.
+# Next steps did not work with a dmg. So creating a sparsebundle is important.
+hdiutil create -size 30g -type "SPARSEBUNDLE" -o "/tmp/$INSTALLER_NAME" -volname "$INSTALLER_NAME" -layout "SPUD" -fs "HFS+J"
```

Comment:
25G is large enough (I think 15G is fine, but only tested down to 25G).

Reply:
Word